audit-triplestore/src/main/java/org/fcrepo/camel/audit/triplestore/AuditSparqlProcessor.java

```diff
@@ -36,29 +62,111 @@
      */
     public void process(final Exchange exchange) throws Exception {
         final Message in = exchange.getIn();
+        final UriRef eventURI = new UriRef("XXX");
```

Comment:
I think for the external events, we should let the user configure a baseURL, and then just add a UUID to the end of it to ensure it's unique.